wiki/Beatmap_Editor_Guides/Compressing_Beatmap_Files/en.md

```diff
@@ -10,12 +13,12 @@ There are 2 types of compression. **Lossy** and **Lossless** compression.
 Zip
 ---
 
-The \*.osz package is actually a \*.zip file, and will compress your beatmap, although not very much, since audio, video, and images are usually compressed very well. Zip and other archives are all lossless compression, otherwise you wouldn't be able to get your data back.
+The `*.osz` package is actually a `*.zip` file, and will compress your beatmap, although not very much, since audio, video, and images are usually compressed very well. Zip and other archives are all lossless compression, otherwise you wouldn't be able to get your data back.
 
 Video
 -----
 
-Recommended reading: [Encoding Video](Encoding_Video "wikilink").
+Recommended reading: [Encoding Video](/wiki/Beatmap_Editor_Guides/Compressing_Videos).
 
 Videos are usually where people go wrong. There are thousands of video codecs out there, and *believe it or not, YouTube is usually a terrible place to get videos*. For the best quality, you should always downsize from a high resolution source (720p YouTube videos are fine for this).
 
```

Comment:
line 27: I suggest to just link to the audio guide and delete the content of this section instead of regurgitating similar parts again